src/client/public-openfin/launcher.json

```diff
@@ -55,7 +55,7 @@
   "snapshot": {
     "windows": [
       {
-        "autoShow": true,
+        "autoShow": false,
```

Comment:
I am puzzled that we have not deemed it necessary to add this in the past .. it must have got measurably worse, but I must admit, I have not noticed any marked change over the time I have been using RT